test/integration/onchain/certificate.test.ts

```diff
@@ -17,13 +17,15 @@ import {
   regulatorAlias,
   regulatorAddress,
 } from '../../helpers/mock'
-import Database, { CertificateRow } from '../../../src/lib/db'
+import Database from '../../../src/lib/db'
+import { CertificateRow } from '../../../src/lib/db/types'
```

Comment:
DB changes as separate PR from API controller, testing first with initiation and issuance?